3515/1/files/gems_get_params.rb

```diff
@@ -0,0 +1,60 @@
+class GemParamGetter
+  attr_reader :contributors
+  attr_reader :forks
+  attr_reader :issues
+  attr_reader :stars
+  attr_reader :used_by
+  attr_reader :watched_by
+  attr_reader :gem_url
+  attr_reader :gem_name
+  attr_reader :params
+
+  def define_variables(gem_url, gem_name)
+    @gem_name = gem_name
+    @gem_url = gem_url
+    @params = return_params_array
+  end
+
+  def return_params_array
+    [gem_name, parse_watched_by, parse_stars, parse_forks, parse_issues,
+     parse_contributors, parse_used_by, gem_rate_count]
+  end
+
+  private
+
+  def html_page(gem_url)
+    @html_page ||= Mechanize.new.get(gem_url)
+  end
+
+  def parse_watched_by
+    html_page(gem_url).css('ul.pagehead-actions a')[1].text.delete('^0-9').to_i
+  end
+
+  def parse_stars
+    html_page(gem_url).css('ul.pagehead-actions a')[3].text.delete('^0-9').to_i
+  end
+
+  def parse_forks
+    html_page(gem_url).css('ul.pagehead-actions a')[5].text.delete('^0-9').to_i
+  end
+
+  def parse_issues
+    html_page(gem_url).css('nav.hx_reponav span')[4].text.to_i
+  end
+
+  def parse_contributors
+    @html_page = nil
```

Comment:
If you get some incorrect data in instance variable and you don't need that data, that means that use it not on purpose and probably you need to separate logic that writes wrong data in it. Or make not instance variable, but just method